validation-test/compiler_crashers_2_fixed/0209-rdar56055600.swift

```diff
@@ -0,0 +1,2 @@
+// RUN: %target-swift-frontend %s -emit-silgen -o /dev/null
+let _: ([Int]) -> Int = \[Int].count
```

Comment:
A FileCheck test in test/SILGen/ might be better since then you can CHECK: that the closure's SIL function has the capture as a parameter.